### ISSUE TITLE
Portrait of person will not look more "amateur" no matter what i do.

### ISSUE THREAD

**Issue by maximilianspitzer** [lllyasviel/Fooocus#1349]
**Read Troubleshoot**

[x] I admit that I have read the [Troubleshoot](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md) before making this issue.

**Describe the problem**
The images i'm creating won't look more amateur like, no matter how i structure my prompt. I want the image to not have as much background blur(depth of field), and not look as if it was professionally taken.

**Full Console Log**
C:\Users\Max\Documents\Image_Generation>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset realistic
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py', '--preset', 'realistic']
Loaded preset: C:\Users\Max\Documents\Image_Generation\Fooocus\presets\realistic.json
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.1.825
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
Total VRAM 10240 MB, total RAM 32684 MB
Set vram state to: NORMAL_VRAM
Disabling smart memory management
Device: cuda:0 NVIDIA GeForce RTX 3080 : native
VAE dtype: torch.bfloat16
Using pytorch cross attention
Refiner unloaded.
model_type EPS
adm 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra keys {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids'}
Base model loaded: C:\Users\Max\Documents\Image_Generation\Fooocus\models\checkpoints\realisticStockPhoto_v10.safetensors
Request to load LoRAs [['SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors', 0.25], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [C:\Users\Max\Documents\Image_Generation\Fooocus\models\checkpoints\realisticStockPhoto_v10.safetensors].
Loaded LoRA [C:\Users\Max\Documents\Image_Generation\Fooocus\models\loras\SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors] for UNet [C:\Users\Max\Documents\Image_Generation\Fooocus\models\checkpoints\realisticStockPhoto_v10.safetensors] with 788 keys at weight 0.25.
Loaded LoRA [C:\Users\Max\Documents\Image_Generation\Fooocus\models\loras\SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors] for CLIP [C:\Users\Max\Documents\Image_Generation\Fooocus\models\checkpoints\realisticStockPhoto_v10.safetensors] with 264 keys at weight 0.25.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.76 seconds
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 3.0
[Parameters] Seed = 3629858835360076716
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 60 - 30
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] no background blur, amateur photograhp, extremely detailed beautiful, stunning, perfect romantic, delicate, pretty, amazing, cinematic, elegant, intricate, epic, highly thought advanced, artistic, best, awesome, surreal, dramatic ambient light, sharp focus, vibrant colors, open, futuristic, fantastic, atmosphere, magical, lovely, complex, cheerful, iconic, cute
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] no background blur, amateur photograhp, extremely detailed, beautiful, stunning, perfect composition, elegant, calm, intricate, cinematic color, highly saturated colors, inspired, brave, amazing, epic, confident, shiny, brilliant, awesome, creative, inspiring, gorgeous, daring, cool, pretty, attractive, smart, elite, best, light, hopeful, vibrant
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.12 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Fooocus] Image processing ...
[Fooocus] VAE encoding ...
Final resolution is (1152, 896).
[Parameters] Denoising Strength = 0.5
[Parameters] Initial Latent shape: torch.Size([1, 4, 144, 112])
Preparation time: 3.37 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 1.2431749105453491
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.28 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 60/60 [00:26<00:00,  2.29it/s]
Image generated with private log at: C:\Users\Max\Documents\Image_Generation\Fooocus\outputs\2023-12-12\log.html
Generating and saving time: 28.42 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 1.2431749105453491
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.26 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 60/60 [00:26<00:00,  2.29it/s]
Image generated with private log at: C:\Users\Max\Documents\Image_Generation\Fooocus\outputs\2023-12-12\log.html
Generating and saving time: 28.55 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.54 seconds
Total time: 64.11 seconds

**Comment by xhoxye** [comment]
Turn off V2 style, and all style
Try this

"prompt": "Snapshot of {prompt} , third-person long-shot, taken from far away, random pic, street, spring, clear blue sky, poorly lit, overexposed, underexposed, poorly composed, low-budget, low-resolution, low-quality, 480p, 240p, color cast, cluttered, shakily taken, early 2000s, late 1990s, vacation, VGA, flash, analog, snapshot, phst artstyle, webcam, bodycam, gopro, indie, found footage, still, harsh natural lighting, ISO noise, bad contrast, JPEG compression artifacts, dynamic angles, bloom, Nokia 6600, cell phone, film camera, vaseline lens, amateur, Facebook, vernacular photography, blurred backgroun",


"negative_prompt": "(closeup:1.5),high resolution, high quality, professional, well lit, well composed, stable, digital, high budget, clear, focused, modern"

![2023-12-12_20-20-44_5267](https://github.com/lllyasviel/Fooocus/assets/129571231/9748aa7e-e153-4053-a1df-9ae17807dc27)

**Comment by hesi18** [comment]
Hi I also tried your prompts but it doesnt help, neither to change sharpness or guidance scale.
Is there anything to have a clear background and not blurred?

**Comment by mashb1t** [comment]
@hesi18 It wopuld be great if you could post your prompt and all parameters / meta data and some screenshots so we can help ypou better. Thanks.

**Comment by mashb1t** [comment]
There are now iphone and spontaneous photo styles available.
Closing as stale, please reopen when providing new information.
